components/form/ConsoleBar.vue

```diff
@@ -0,0 +1,90 @@
+<script>
+import ButtonDropdown from '@/components/ButtonDropdown';
+
+export default {
+  name: 'ConsoleBar',
+
+  components: { ButtonDropdown },
+
+  props: {
+    resource: {
+      type:     Object,
+      required: true,
+      default:  () => {
+        return {};
+      }
+    },
+  },
+
+  data() {
+    return { };
+  },
+
+  computed: {
+    isDown() {
+      return this.isEmpty(this.resource);
+    },
+
+    isRunning() {
+      return this.resource.actualState === 'Running';
```

Comment:
Can we use the property `isRunning` of the vm's model to check?

https://github.com/harvester/dashboard/blob/ae7a3caf7fedcb3fb2bf3cabfe583d86781057c1/models/kubevirt.io.virtualmachine.js#L420

Reply:
Done.